include/crm/common/internal.h

```diff
@@ -89,7 +89,7 @@ pcmk__xe_set_bool_attr(xmlNodePtr node, const char *name, bool value);
  *         the value "true", False in all other cases
  */
 bool
-pcmk__xe_attr_is_true(xmlNodePtr node, const char *name);
+pcmk__xe_attr_is_true(const xmlNode *node, const char *name);
```

Comment:
Do we prefer `xmlNode *` over `xmlNodePtr`? I was just looking at this a few minutes ago in a different context and found the following, which goes the opposite direction: https://github.com/ClusterLabs/pacemaker/commit/cd098b06db263cfb337d4b8eaf2724df66fff060

Reply:
const xmlNodePtr != const xmlNode *

It's a quirk of C. The former doesn't work as expected.

Reply:
libxml2 itself uses const xmlNode * for function signatures that require it

Reply:
That is bizarre. But so are a lot of things we've come to take for granted in language specs. Good to know.

Reply:
GListPtr was just something we'd made up, but it looked like something out of GLib.  To be less confusing, we deprecated it and stopped using it everywhere.  xmlNodePtr is at least provided by libxml.  I did initially add more uses of it throughout the code, but I don't really care one way or the other which we use.

I think in general, most people would say you shouldn't typedef away the pointer, so maybe we shouldn't use it.